crates/oxc_linter/src/snapshots/eslint_no_misleading_character_class.snap

```diff
@@ -0,0 +1,776 @@
+   ·          ──────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected emoji modifier in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[a\uD83C\uDFFB]/u
+   ·          ───────────────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected emoji modifier in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[\uD83D\uDC76\uD83C\uDFFB]/u
+   ·          ──────────────────────────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected emoji modifier in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[\u{1F476}\u{1F3FB}]/u
+   ·          ────────────────────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected surrogate pair in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[🇯🇵]/
+   ·          ────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected surrogate pair in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[🇯🇵]/i
+   ·          ────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected regional indicator in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[🇯🇵]/u
+   ·          ────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected regional indicator in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[\uD83C\uDDEF\uD83C\uDDF5]/u
+   ·          ──────────────────────────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected regional indicator in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[\u{1F1EF}\u{1F1F5}]/u
+   ·          ────────────────────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected joined character sequence in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[👨‍👩‍👦]/
+   ·          ────────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected surrogate pair in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[👨‍👩‍👦]/
+   ·          ────────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected joined character sequence in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[👨‍👩‍👦]/u
+   ·          ────────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected joined character sequence in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[👩‍👦]/u
+   ·          ──────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected joined character sequence in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[👩‍👦][👩‍👦]/u
+   ·          ────────────
+   ╰────
+
+  ⚠ eslint(no-misleading-character-class): Unexpected joined character sequence in character class.
+   ╭─[no_misleading_character_class.tsx:1:10]
+ 1 │ var r = /[👨‍👩‍👦]foo[👨‍👩‍👦]/u
+   ·          ───────────────────
```

Comment:
this span looks funky? it's longer than the ast node.

Reply:
this might cause a panic when diagnostic is generated (probably not since these test snapshots were ok to be generated